.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+env:
+  CARGO_TERM_COLOR: always
+jobs:
+  rustfmt:
+    name: rustfmt
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Check formatting
+      run: cargo fmt --all -- --check
+
+  test:
+    name: test
+    env:
+      # Emit backtraces on panics.
+      RUST_BACKTRACE: 1
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Build crate
+      run: cargo build --verbose
+    # This step helps separate buildtime and runtime errors in tests.
+    - name: Build tests
+      run: cargo build --verbose --tests
+    - name: Run Tests
```

Comment:
```suggestion
```

Minus doesn't have any tests and I don't think test will help it much because, in order to quit the program, the user has to press q. In CI, there isn't anyone for doing that. The test would run infinitely. The only way we can test it is probably by human checks